firmware/rhizome.ino

```diff
@@ -80,5 +90,13 @@ void loop() {
  * Delegation function that allows us to call the managed method for publishing periodic updates.
  */
 void doPeriodicUpdates() {
-    sproutList.publishPeriodicUpdates();
+	//Do not attempt to publish updates if disconnected from the cloud
+    if(Particle.connected() == false){
```

Comment:
If `Particle.connected()` returns a boolean, you should just write this as `!Particle.connected()`